src/main/java/seedu/duke/ui/Ui.java

```diff
@@ -54,6 +55,9 @@
             + LS
             + "fitness and health. Please type a command or view the list of "
             + "available commands by typing " + HelpCommand.MESSAGE_COMMAND_FORMAT + ".";
+    public static final String MESSAGE_ERROR_PROFILE_NOT_CREATED = "Profile has not yet been created.\n"
```

Comment:
I think instead of asking user to type command to create a profile, we can directly help him create a profile, then we ask the user to fill in his name, weight, height and calorie goal, maybe like:
```
You haven't created a profile, a new profile will be created to save personal data:
Name: (wait for user input)
Weight: (wait for user input)
Height: (wait for user input)
Calorie goal: (wait for user input)
```
In the same time can tell the user if have no idea can just press enter, then you handle when "" is detected, initialized to default value :)

Reply:
Great idea! Let me think about how to implement it.

Reply:
Yep! I think we would also need them to key in their Age too as well as a selection process for their Basal Metabolic Rate, as there's like different activity factor to consider in our calculations for calories burnt.
Theres like 5 modes of BMR so I was thinking maybe you will reach a stage where it will show:

1. Sedentary - Little or no exercise
2. Lightly Active - Light exercise or sports, around 1-3 days a week
3. Moderately Active - Regular exercise or sports, around 3-5 days a week
4. Very Active - Frequent exercise or sports, around 6-7 days a week
5. If you are extra active - Sports or exercising is your passion and a physical jobscope (I think can omit this if its too extra)

and any inputs from 1-5 lets you select your activity factor. (Pass their input to me can already then I will process it)
If have any other thoughts feel free to discuss with me ah but I think that's about it for setting up the profile :) 
Thanks Yi Zhi!